docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+# Contributing to `ITKRemoteModuleBuildTestPackageAction`
+
+The Insight Toolkit (ITK) open-source community is driven by
+involvement from users and developers alike. Contributions
+can include new features, bug fixes, or simply
+documentation updates. This document describes the community
+guidelines to help contributors (like you!) improve the
+`ITKRemoteModuleBuildTestPackageAction` repository.
+
+Check out the project [README](../README.md)
+before reading this to better understand the purpose and components
+of the `ITKRemoteModuleBuildTestPackageAction` repository.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Creating or Modifying a Workflow](#creating-or-modifying-a-workflow)
+- [Testing Your Changes](#testing-your-changes)
+- [Creating a Pull Request](#creating-a-pull-request)
+
+## Prerequisites
+
+In order to contribute to `ITKRemoteModuleBuildTestPackageAction` you
+must have the following:
+- A [GitHub](https://github.com/) account;
+- A PC with a [git](https://git-scm.com/) client and a text editor of your choice.
+
+The following are also helpful, but not strictly required:
+- A PC running the operating system for which your changes are targeted,
+  i.e. Windows, Ubuntu (Linux), or MacOS. This will allow you to test changes locally
+  before pushing to GitHub.
+- A Python interpreter of 3.7 or greater for testing wheels locally.
```

Comment:
Python 3.7 EOL is expected within 6 months. We may start thinking of deprecating it.